src/main/java/mat/server/service/impl/MatContextServiceUtil.java

```diff
@@ -205,7 +221,7 @@ private boolean canUserConvert(String ownerId) {
         return isOwner || isSuperUser;
     }
 
-    private boolean isVersionEligibleForConversion(String qdmVersion, String matVersion) {
+    private boolean isVersionEligibleForFhirValidation(String qdmVersion, String matVersion) {
         return QDM_VER_BEFORE_CONVERSION.compareTo(asDecimalVersion(qdmVersion)) <= 0
                 && MAT_VER_BEFORE_CONVERSION.compareTo(asDecimalVersion(matVersion)) <= 0;

```

Comment:
Instead of the class constants, could you the properties in MAT.properties for this comparison?
`mat.measure.current.release.version=v6.0`
and
`mat.measure.current.qdm.version=5.5`

Reply:
No. This is different. Those are threshold versions, not the current versions.

Reply:
Ah, ok. Thanks for clarifying!